gatling-engine/src/main/scala/io/snice/gatling/requests/Ss7Requests.scala

```diff
@@ -10,4 +10,7 @@ object Ss7Requests {
     ss7("purgeMs")
       .pur("${imsi}")
 
+  val air: Ss7RequestBuilder =
+    ss7("authenticationInfo")
+      .air("${imsi}", "${airNumberOfVectors}")
```

Comment:
would be nice to have a more fluent API here instead of this (in case we want to add more params in the future). 

you could basically define multiple specific methods to add your key/value to the map when you build the AIR and so.
check here https://github.com/sniceio/snice-gatling/blob/main/gatling-diameter/src/main/scala/io/snice/gatling/diameter/request/builder/DiameterRequestBuilder.scala#L111

the result is something like this which is pretty nice and easy to extend to use https://github.com/sniceio/snice-gatling/blob/main/gatling-engine/src/main/scala/io/snice/gatling/requests/AuthenticationInfoRequest.scala#L25